README.md

```diff
@@ -139,6 +141,13 @@ on all responses served by ecstatic.
 Set `opts.contentType` in order to change default Content-Type header value.
 Defaults to **application/octet-stream**.
 
+### `opts.mimeTypes`
+
+Add new or override one or more mime-types. This affects the HTTP Content-Type header.
+Can either be a path to a [`.types`](http://svn.apache.org/repos/asf/httpd/httpd/trunk/docs/conf/mime.types) file or an object hash of type(s).
+    
+    ecstatic({ mimeType: { 'mime-type': ['file_extension', 'file_extension'] } })
```

Comment:
Does this kins of doc make sense? An example would be `ecstatic({ mimeType: { 'node/ecstatic+fun': ['ecs', 'fun'] } })` to get `content-type: node/ecstatic+fun; charset=UTF-8` with `.esc` and `.fun` files.